### PR TITLE
Minor/improve healthchecks

### PR DESCRIPTION
**Pull request by coltmcnealy-lh**
Refactors health checks:
 - Creates HealthService.java which has a Javalin service Prometheus metrics and also healthchecks
 - Adds a readiness check which tells whether the service can answer requests
 - Adds a liveness check which tells whether the service should be restarted
 
The readiness and liveness checks run on the same Javalin server as the Prometheus exporter.

### CODE REVIEW

File: docs/SERVER_CONFIGURATIONS.md [hunk 1]
```diff
@@ -561,20 +561,40 @@ creating the `ExternalEventDef`.
...
+The port that the healthchecks and prometheus metrics are exposed on.
 
 - **Type:** int
 - **Default:** 1822
 - **Importance:** low
 
 ---
 
-### `LHS_PROMETHEUS_EXPORTER_PATH`
+### `LHS_HEALTH_PATH_PROMETHEUS`
```
Comment (sauljabin) [inline docs/SERVER_CONFIGURATIONS.md:574]: LHS_HEALTH_PATH_PROMETHEUS maybe LHS_HEALTH_PATH_METRICS?

File: server/src/main/java/io/littlehorse/server/metrics/PrometheusMetricExporter.java [hunk 2]
```diff
@@ -69,19 +62,10 @@ public void bind(Map<String, KafkaStreams> streams) {
...
-        log.info("Prometheus started");
-    }
-
-    private Handler handleRequest() {
+    public Handler handleRequest() {
         return ctx -> {
-            log.debug(
-                "Request [from={}, path={}, method={}]",
+            log.trace(
+                "Metrics Request [from={}, path={}, method={}]",
```
Comment (sauljabin) [inline server/src/main/java/io/littlehorse/server/metrics/PrometheusMetricExporter.java:68]: we agreed on removing this log injection

**Review by sauljabin — APPROVED** [review]

File: server/src/main/java/io/littlehorse/server/metrics/PrometheusMetricExporter.java (hunk 2 above)
Comment (coltmcnealy-lh) [reply server/src/main/java/io/littlehorse/server/metrics/PrometheusMetricExporter.java:68]: Good catch, I forgot to push the change.

File: docs/SERVER_CONFIGURATIONS.md (hunk 1 above)
Comment (coltmcnealy-lh) [reply docs/SERVER_CONFIGURATIONS.md:574]: Ok I can change it. Will change on the Operator too.

**Pull request merged**